src/dashboard/src/settings/common.py

```diff
@@ -202,20 +204,17 @@
                 'django.template.context_processors.static',
                 'django.template.context_processors.request',
                 'django.contrib.messages.context_processors.messages',
-                'shibboleth.context_processors.login_link',
-                'shibboleth.context_processors.logout_link',
             ],
             'debug': DEBUG,
         },
     },
 ]
 
-AUTHENTICATION_BACKENDS = (
+AUTHENTICATION_BACKENDS = [
```

Comment:
These are changed to square brackets to make modifications easier in the Shibboleth section (Later versions of Django have switched over to doing this by default in the settings file anyway)